dist/README.md

```diff
@@ -36,25 +36,32 @@ You can grab the relevant MathJax files in `./dist/extras/mathjax/`.
 ### To include localization
 
 Plotly.js defaults to US English (en-US) and includes British English (en) in the standard bundle.
-Many other localizations are available - here is an example using Swiss-German (de-CH),
-see the contents of this directory for the full list.
-They are also available on our CDN as https://cdn.plot.ly/plotly-locale-de-ch-latest.js OR https://cdn.plot.ly/plotly-locale-de-ch-1.38.1.js
-Note that the file names are all lowercase, even though the region is uppercase when you apply a locale.
+This example uses Swiss-German (de-CH), but many other localizations are available (see the contents of this directory for the full list). All available locales have working date localizations, but only a subset will localize on-graph text (e.g., modebar controls). See [here](https://github.com/plotly/plotly.js/pulls?q=is%3Apr+label%3A%22type%3A+translation%22+is%3Aclosed) for a list of "fully-supported" locales.
 
-*After* the plotly.js script tag, add:
+#### Loading and registering locales
+
+Some locales build on other locales (e.g., "de-CH" builds on "de"). In this case, first load the base locale (e.g., "de"), then load the actual locale (e.g., "de-CH"):
 
 ```html
+<script src="plotly-locale-de.js"></script>
```

Comment:
I think this section is misleading. a priori `de-CH` doesn't _depend_ on `de`. At the moment, `de-CH` fallbacks to `de` just because `de-CH` isn't "fully" supported.

Reply:
So, I propose we use a "fully" supported language for this example and invite folks we want to use a non-fully supported localization (e.g. `de-CH`) to contribute by making a PR.